cloudinit/util.py

```diff
@@ -184,6 +184,27 @@ def fully_decoded_payload(part):
     'system-version': 'product_version',
 }
 
+# FreeBSD's kenv and dmidecode use different names for the same value,
+# this allows us to refer to them by one cannonical name
+DMIDECODE_TO_KENV_MAPPING = {
```

Comment:
reading this, I thought "why is util.py littered with dmi information".

So... https://aci-github.cisco.com/atom/atomix/pull/1269

Reply:
well, the file is called util.py
so it's just a collection of stuff, until you realise, this cluster of stuff should be its own module / class / etc